howtoplay.html

```diff
@@ -24,7 +24,7 @@
       <div class="header">
         <img class="play-button" src="images/docs/play-button.png" alt="Play" />
         <h1 class="site_title centered lang" data-href="how">HOW TO PLAY</h1>
-        <a href="https://github.com/fossasia/labyrinth">
+        <a href="">

```

Comment:
Maybe you should link this to `index.html` or `#` instead.

Reply:
Yes. Point it to `#`. That'll be good.